flume-ng-dist/src/main/appended-resources/META-INF/NOTICE

```diff
@@ -217,8 +217,8 @@ source: https://www.opensource.apple.com/source/configd/configd-453.19/dnsinfo/d
 Scala
 scala-collection-compat
 scala-java8-compat
-Copyright (c) 2002-2022 EPFL
-Copyright (c) 2011-2022 Lightbend, Inc.
+Copyright (c) 2002-2023 EPFL
+Copyright (c) 2011-2023 Lightbend, Inc.
```

Comment:
just to confirm, did we update this dependency to a version that changed its copyright year to include 2023?

Reply:
This refers to the date of the citation, which I don't think needs to be updated to 2023.